### PR TITLE
bugfix/1226/fixing-reply-comment-translation-bug

### PR DESCRIPTION
**Pull request by DeLettenhove**
Fixed bug with error message translation when you change language of the app

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| Original screenshot | Updated screenshot 1 Updated screenshot 2|

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue #1226

### CODE REVIEW

File: src/locales/en/error.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
   "onlyLetter": "Field should contain only letters",
   "emailLength": "Field should contain from 8 to 60 characters",
   "wrongFormat": "Wrong format",
+  "text": "Field should contain from 2 to 700 characters",
```
Comment (maarekua) [inline src/locales/en/error.json:12]: The naming of the constants and object keys should be informative, "text" is not. Change it to describe what is behind this translation.

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/locales/en/error.json (hunk 1 above)
Comment (DeLettenhove) [reply src/locales/en/error.json]: I fixed this.

**Review by maarekua — APPROVED** [review]

File: src/pages/product-details/comments/comments-item/reply-form/reply-form.js [hunk 2]
```diff
@@ -72,7 +71,7 @@ const ReplyForm = ({ cancel, commentId, refetchComments }) => {
           onChange={handleCommentChange}
           onBlur={handleBlur}
           error={!!errors.text}
-          helperText={errors.text || ''}
+          helperText={errors.text ? t('error.replyCommentField') : ''}
```
Comment (ElitKondor) [inline src/pages/product-details/comments/comments-item/reply-form/reply-form.js:74]: You don't need to use ternary operator when else statement returns nothing. Just use '&&' operator instead which is better approach in this case

File: src/locales/ua/error.json [hunk 3]
```diff
@@ -9,6 +9,7 @@
   "onlyLetter": "Поле повинно містити лише літери",
   "emailLength": "Поле повинно містити від 8 до 60 символів",
   "wrongFormat": "Некоректний формат",
+  "replyCommentField": "Поле повинно містити від 2 до 700 символів",
```
Comment (ElitKondor) [inline src/locales/ua/error.json:12]: I found the same translation inside Bohdan's (Yavdoshniak) pull request. Please pull changes and delete unnecessary fields

**Comment by Skrekliam** [comment]
Task is done already

**Pull request closed without merging**